script/deploy/4_VerifyOFT.s.sol

```diff
@@ -2,32 +2,29 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.24;
 
-import {BaseScript, PeerConfig, ReceiveLibConfig, SendLibConfig} from "./BaseScript.s.sol";
-import {BatchScript} from "./BatchScript.s.sol";
+import {BaseScript, ChainDeployment, PeerConfig, ReceiveLibConfig, SendLibConfig} from "../BaseScript.s.sol";
+import {BatchScript} from "../BatchScript.s.sol";
 import {
     ILayerZeroEndpointV2,
     IMessageLibManager
 } from "@layerzerolabs/lz-evm-protocol-v2/contracts/interfaces/ILayerZeroEndpointV2.sol";
 
 import {L2YnERC20Upgradeable} from "@/L2YnERC20Upgradeable.sol";
 import {L2YnOFTAdapterUpgradeable} from "@/L2YnOFTAdapterUpgradeable.sol";
-import {ImmutableMultiChainDeployer} from "@/factory/ImmutableMultiChainDeployer.sol";
 
 import {IOAppCore} from "@layerzerolabs/lz-evm-oapp-v2/contracts/oapp/interfaces/IOAppCore.sol";
 import {RateLimiter} from "@layerzerolabs/lz-evm-oapp-v2/contracts/oapp/utils/RateLimiter.sol";
 import {TimelockController} from "@openzeppelin/contracts/governance/TimelockController.sol";
 import {ProxyAdmin} from "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
 import {console} from "forge-std/console.sol";
 
-// forge script script/VerifyL2OFTAdapter.s.sol:DeployL2Adapter \
-// --rpc-url ${rpc} --sig "run(string calldata)" ${path} \
-// --account ${deployerAccountName} --sender ${deployer}
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
 
-contract VerifyL2OFTAdapter is BaseScript, BatchScript {
-    ImmutableMultiChainDeployer public multiChainDeployer;
-    L2YnOFTAdapterUpgradeable public l2OFTAdapter;
-    L2YnERC20Upgradeable public l2ERC20;
+// forge script VerifyOFT --rpc-url ${rpc} \
+// --sig "run(string calldata,string calldata)" ${input_path} ${deployment_path} \
+// --account ${deployerAccountName} --sender ${deployer}
 
+contract VerifyOFT is BaseScript, BatchScript {
```

Comment:
Consolidated `VerifyL1OFTAdapter` and `VerifyL2OFTAdapter` scripts into a single `VerifyOFT` script